packages/rudy/src/core/createRouter.js

```diff
@@ -98,7 +98,6 @@ export default (
     // TODO: Fix these annotations
     Object.assign(api, { getLocation, dispatch, getState })
 
-    getState.rudy = api // make rudy available via `context` with no extra Providers, (see <Link />)
```

Comment:
This is a hack, but also it doesn't work with react-redux 6+ because the store is no longer in the context for the Link component to access.

Reply:
Wait so did we lose the ability to expose our internal api or do we just change how we access it

Reply:
Changed how we access it. Basically instead of monkey patching Redux, we now use React context to pass down the API through the component tree. You wrap your component tree in `<RudyProvider />`, then `<Link />` and `<NavLink />` can access the API. (Or alternatively, `<RudyConsumer />` is available to build a component taht uses the API).

Reply:
@faceyspacey some reading on this process. Might help us get around the monkey patch in redux (on respond)